Martium.TravelInfo.App/Forms/TravelInfoForm.cs

```diff
@@ -29,9 +30,9 @@ public TravelInfoForm()
 
         private void TravelInfoForm_Load(object sender, EventArgs e)
         {
-            LoadTravelInfoSettings();
+            LoadCountryIsoCollections();
```

Comment:
rename `LoadCountryIsoCollections` to `LoadDepartureCountryConmboBox` (method should be named from user perspective not tech details)